html2asketch/nodeToSketchLayers.js

```diff
@@ -77,6 +77,13 @@ function fixBorderRadius(borderRadius) {
 
 export default async function nodeToSketchLayers(node) {
   const layers = [];
+
+  // Ignore node if one of its ancestors is hidden, or if it has a fixed position
+  // https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent
+  if (node.offsetParent === null) {
```

Comment:
yeah, lets move it below (where all `if(…){return layers}` are) and bail out only if position `!==` fixed